### ISSUE TITLE
keycloak user in keystone incorrect UUID format

### ISSUE THREAD

**Issue by mpiscaer** [vexxhost/atmosphere#673]
The current keystone image with hash 408b84f8e1bb237dfe8b84d52b5d77fd1c307b2538e77d7e2ab69ecb74385886 gives the user in a incorrect UUID format:
```shell
root@keycloak:~# openstack user list --domain atmosphere
+----------------------------------+------------+
| ID                               | Name       |
+----------------------------------+------------+
| 2bb15b5106184c7ea639a9f798223600 | test-user  |
| a0976e5fbd9844f2a66034673f7963fc | test-user2 |
+----------------------------------+------------+
```

The keystone with image ID: 4c63257d75c72137454690ad8cef600742e20b07ab31d873b88fa382ce659d17 gives the correct UUID:
```shell
root@keycloak:~# openstack user list --domain atmosphere
+--------------------------------------+------------+
| ID                                   | Name       |
+--------------------------------------+------------+
| 2bb15b51-0618-4c7e-a639-a9f798223600 | test-user  |
| a0976e5f-bd98-44f2-a660-34673f7963fc | test-user2 |
+--------------------------------------+------------+
```

**Comment by mnaser** [comment]
Is this necessary problematic?  I think this is mainly because of https://github.com/vexxhost/atmosphere/issues/571 which @etimmons-sp solved.

**Comment by mpiscaer** [comment]
@mnaser I don't know if it is problematic for openstack services if at one point the UUID changes. But mainly the following CI https://github.com/vexxhost/atmosphere/blob/main/molecule/keycloak/verify.yml#L53 failes there they are not the same anymore.

**Comment by mnaser** [comment]
Oh interesting, we should probably either update the CI or address this.   I'd be curious to hear what @etimmons-sp has to say about this.

**Comment by etimmons** [comment]
The first user table with the new keystone image is how the users' uuids should look.  This is a result of normalizing the UUIDs to keystone's preference in the updated keystone-keycloak-backend driver (1.6):

https://github.com/vexxhost/keystone-keycloak-backend/blob/main/keystone_keycloak_backend/identity.py#L73

The second table is the old way of how the driver would pass back the uuid to keystone for entry into that table.

**Comment by mpiscaer** [comment]
@etimmons @mnaser So https://github.com/mpiscaer/atmosphere/commit/e69b9f62bdc4b487cd3a1d314e6541f506bd50f2 would correct the CI, wasn't sure if this the correct way so I reverted the commit.

What do you thing both would this be a correct sollution?

**Comment by mpiscaer** [comment]
I have created a PR for this https://github.com/vexxhost/atmosphere/pull/677

**Comment by mnaser** [comment]
Fixed by #677
